methods/view_metabolic_model_details/display.yaml

```diff
@@ -0,0 +1,47 @@
+#
+# Define basic display information
+#
+name     : View Metabolic Model Details
+subtitle : |
+    Bring up a detailed view of your metabolic model within the narrative. [7]
```

Comment:
should say "Bring up a detailed view of a metabolic model". (Not sure how to edit in the PR.)